db/migrate/20171104221735_create_passwordless_sessions.rb

```diff
@@ -13,7 +13,7 @@ def change
       t.datetime :claimed_at
       t.text :user_agent, null: false
       t.string :remote_addr, null: false
-      t.string :token, null: false
+      t.string :token_digest, null: false
```

Comment:
Does it make sense to modify the existing migration, or should we add a second migration?

Reply:
I think we should modify the original but provide upgrade instructions for existing installations.